vtysh/vtysh.h

```diff
@@ -44,6 +44,7 @@ DECLARE_MGROUP(MVTYSH);
 #define VTYSH_FABRICD   0x20000
 #define VTYSH_VRRPD     0x40000
 #define VTYSH_PATHD     0x80000
+#define VTYSH_PIM6D     0x200000
```

Comment:
Why not 0x100000?

Reply:
This is going to be used by MGMTD. https://github.com/FRRouting/frr/pull/10000

Reply:
I don't think we should consider things that aren't in master yet here. (And it will create a merge conflict anyway.)

Reply:
Sure will change this